### PR TITLE
style- button styling

### PR DESCRIPTION
**Pull request by ImrozNezami**
Different buttons
Primary button of different size
Primary button with icon
Success button
Warning button
Alert button
Secondary button
Icon button
Link button
Float button
Screenshot (215)

### CODE REVIEW

File: button.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+.md-primary {
+  font-size: 1.2rem;
+  padding: 1.3rem;
+}
+.lg-primary {
+  width: 70%;
+}
+.btn-float {
+  width: 70px;
```
Comment (SimranjitKaurMaan) [inline button.css]: Prefer to use `rem` instead of px.

**Comment by SimranjitKaurMaan** [comment]
Also, button corners should be rounded a bit as sharp corners are not good to see.

File: button.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+.btn:active {
+  border: 2px solid;
+  border-color: var(--alertblue);
+}
+.btn-primary {
+  background-color: var(--primary);
+}
+.font-sm {
+  font-size: 1rem;
+}
```
Comment (kjais1720) [inline button.css]: This class should be in the utility stylesheet since it will be used across many components and not just this one

**Review by kjais1720 — COMMENTED** [review]
All good, just one suggestion, try to use rems as much as possible and pixels as less as possible

File: button.css (hunk 1 above)
Comment (ImrozNezami) [reply button.css]: Done

**Comment by ImrozNezami** [comment]
> Also, button corners should be rounded a bit as sharp corners are not good to see.

Done

File: button.css (hunk 2 above)
Comment (ImrozNezami) [reply button.css]: Done

**Pull request merged**